ansible/roles/debops.docker_registry/defaults/main.yml

```diff
@@ -0,0 +1,629 @@
+#
+# Path where the Docker Registry upstream repository will be checked out.
+docker_registry__git_dest: '{{ docker_registry__gopath + "/src/"
+                               + docker_registry__git_repo.split("://")[1] }}'
+
+                                                                   # ]]]
+# .. envvar:: docker_registry__git_dir [[[
+#
+# Path where the Docker Registry :file:`.git/` directory will be stored.
+docker_registry__git_dir: '{{ docker_registry__src + "/" + docker_registry__git_repo.split("://")[1] }}'
+
+                                                                   # ]]]
+# .. envvar:: docker_registry__git_gpg_key [[[
+#
+# The fingerprint of the GPG key used to sign Docker Registry upstream
+# releases.
+docker_registry__git_gpg_key: '8C7A 111C 2110 5794 B0E8  A27B F58C 5D0A 4405 ACDB'
+
+                                                                   # ]]]
+# .. envvar:: docker_registry__git_repo [[[
+#
+# The URL of the upstream Docker Registry :command:`git` repository.
+docker_registry__git_repo: 'https://github.com/docker/distribution'
+
+                                                                   # ]]]
+# .. envvar:: docker_registry__git_version [[[
+#
+# The branch/tag of the Docker Registry :command:`git` repository to check out.
+docker_registry__git_version: 'v2.7.1'
+
+                                                                   # ]]]
+# .. envvar:: docker_registry__binary [[[
+#
+# The absolute path where the Docker Registry upstream binary will be installed
+# to; alternatively, the path of the Docker Registry binary installed by the
+# APT package.
+docker_registry__binary: '{{ "/usr/local/bin/docker-registry"
+                             if docker_registry__upstream|bool
+                             else "/usr/bin/docker-registry" }}'
+                                                                   # ]]]
+                                                                   # ]]]
+# Web server configuration [[[
+# ----------------------------
+
+# .. envvar:: docker_registry__fqdn [[[
+#
+# The Fully Qualified Domain Name of the Docker Registry service.
+docker_registry__fqdn: 'registry.{{ docker_registry__domain }}'
+
+                                                                   # ]]]
+# .. envvar:: docker_registry__domain [[[
+#
+# The DNS domain used as a base for the Docker Registry FQDN address.
+docker_registry__domain: '{{ ansible_local.core.domain
+                             if (ansible_local|d() and ansible_local.core|d() and
+                                 ansible_local.core.domain|d())
+                             else (ansible_domain if ansible_domain else ansible_hostname) }}'
+
+                                                                   # ]]]
+# .. envvar:: docker_registry__backend_port [[[
+#
+# The TCP port on which the Docker Registry backend service listens for
+# connections.
+docker_registry__backend_port: '5070'
```

Comment:
I tend to avoid using ports suggested by upstreams that are not officially registered - usually it's something like `4000`, `9000`, etc. The '5070' port seems to be available in the `/etc/services` database, so I went with it.

Reply:
Ok, that's an argument I did not looked at. The port number itself is for me irrelevant, I was just confused about the different numbers.